coprocess.go

```diff
@@ -102,7 +102,10 @@ func (c *CoProcessor) ObjectFromRequest(r *http.Request) *coprocess.Object {
 
 	object.HookType = c.HookType
 
-	object.Metadata = make(map[string]string)
+	object.Metadata = map[string]string{
+		"RequestURI": r.RequestURI,
```

Comment:
Does it needed for NewRelic integration somehow, or for your internal purpose?

Reply:
AFAIK we use this in our events.
If it's possible, it would be nice to leave this as it is.